biome.json

```diff
@@ -55,7 +55,7 @@
             },
             "correctness": {
                 "noUnnecessaryContinue": "off",
-                "noUnusedImports": "error"
+                "noUnusedImports": "off"
```

Comment:
Why does this have to be off?

Reply:
Ahh, I had turned this off when I was having issues with Biome. I posted in Slack- it was a weird one! Brandon suggested trying turning it off: https://cscie-39.slack.com/archives/C074PV73PGB/p1733441617082049